build.gradle

```diff
@@ -1,11 +1,16 @@
 // Gradle plugins
 buildscript {
     repositories {
+        mavenLocal()
```

Comment:
I know, it was just used for development, I forgot to remove it.

> On Jan 11, 2016, at 6:09 PM, Minecrell <notifications@github.com> wrote:
> 
> In build.gradle:
> 
> > @@ -1,11 +1,16 @@
> >  // Gradle plugins
> >  buildscript {
> >      repositories {
> > +        mavenLocal()
> If it is then it's broken because that will never work on the Jenkins nor Travis ;)
> 
> —
> Reply to this email directly or view it on GitHub.
>